dascore/transform/differentiate.py

```diff
@@ -60,6 +94,11 @@ def differentiate(
     order
         The order of the differentiation operator. Must be a possitive, even
         integar.
+    step
+        The number of columns/rows to skip for differention.
+        eg: an array of [a b c d e] uses b and d to calculate diff of c when
+        step = 1 and order = 2. When step = 2, a and e are used to calcualte
+        diff at c.
```

Comment:
So, when `step=1` and `order=2` (same as how the `Patch.velocity_to_strain_rate` function used to work before), does it mean technically `gauge_length = 2 * distance step`?

Reply:
Yes, you are right. The gauge length should be 2*dx in the original function.